jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -253,6 +256,13 @@ class SetupAuthenticationDialog(
         }
 
         close(OK_EXIT_CODE)
+        SsoSessionConfigurationManager().writeSsoSessionProfileToConfigFile(
+            state.idcTabState.region.id,
+            state.idcTabState.startUrl,
+            scopes,
```

Comment:
if we need role access, this is not the right set of scopes

Reply:
Sure, I'll look into it and fix it.